Calculator.Logic/CalculatorValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Calculator.Logic
+{
+    public class CalculatorValidator: ICalculatorValidator
+    {
+        public void ValidateOperation(Operation operation)
+        {
+            if (operation == Operation.None)
+            {
+                throw new CalculatorException(Error.None);
+            }
+        }
+
+        public void ValidateTermsForDivision(List<double> terms)
+        {
+            if (terms.Count < 2)
```

Comment:
This is the same as the method below, just call that one instead of duplicating code